packages/design-system/src/components/locale/en.json

```diff
@@ -49,22 +49,26 @@
     "check": "Checkmark",
     "close": "Close",
     "download": "Download",
+    "email": "Email",
     "externalLink": "This link goes to an external site",
     "hhsLogo": "Department of Health and Human Services",
     "image": "Image",
     "infoCircle": "Information",
+    "link": "Link",
     "lockCircle": "Lock in circle",
     "lock": "Lock",
     "menu": "Menu Icon",
     "next": "Next",
     "print": "Print",
     "remove": "Remove",
+    "rss": "RSS",
     "star": "Star",
     "starEmpty": "Empty Star",
     "starFilled": "Filled Star",
     "starHalf": "Half Star",
     "usaLogo": "USA.gov - Government Made Easy",
-    "warning": "Warning"
+    "warning": "Warning",
+    "whatsNew": "What's New"
```

Comment:
nit: replace this quote with one that sparks joy for typography enthusiasts
```suggestion
    "whatsNew": "What’s New"
```

Reply:
@kim-cmsds I love the idea of sparking joy for typography enthusiasts. Will make the change.

Reply:
@kim-cmsds, I made the change to use a curly apostrophe (’), but I'm noticing a tiny box appearing around it in vs code. I'm not sure what this signifies or if it indicates a potential issue with the character encoding? 
<img width="264" alt="Screenshot 2024-08-14 at 12 51 00 PM" src="https://github.com/user-attachments/assets/8e461e62-0178-46e4-9503-2dd27435155d">

Reply:
That same curly apostrophe is used here:
https://github.com/CMSgov/design-system/blob/0fb69bcaa6f1505e09db7c00349c939b1c37222f/packages/design-system/src/components/locale/en.json#L108

Reply:
@kim-cmsds: updated to a curly apostrophe in this json file, as well as `icons.stories.tsx`